src/density_interface.jl

```diff
@@ -1,4 +1,4 @@
-@inline DensityInterface.hasdensity(::Distribution) = true
+@inline DensityInterface.DensityKind(::Distribution) = HasDensity()
```

Comment:
I think it's not exported, is it?
```suggestion
@inline DensityInterface.DensityKind(::Distribution) = DensityInterface.HasDensity()
```

Reply:
Hm, it actually *is* exported:

https://github.com/JuliaMath/DensityInterface.jl/blob/e71eb2ded6f3320d66a6d7efc615e3e60dddb58f/src/interface.jl#L69

Reply:
Ah, but we "import DensityInterface" :-)